src/query/FindOneArgs.ts

```diff
@@ -1,7 +1,12 @@
-import type { WhereQuery } from './WhereQuery';
+import type { WhereQuery, WhereQueryTyped } from './WhereQuery';
 
 export interface FindOneArgs {
   select?: string[];
   where?: WhereQuery;
   sort?: string[] | string;
 }
+
+export interface FindOneArgsTyped<T> extends FindOneArgs {
+  select?: (string & keyof T)[];
+  where?: WhereQueryTyped<T>;
```

Comment:
This is missing the sort option. Is that intended?

Reply:
It's extending FindOneArgs, which has it. I can't really get fancy with types since it takes stuff like ```'column_name ASC'```